src/components/Counter.js

```diff
@@ -3,37 +3,47 @@ import { connect } from 'react-redux';
 import { increment, decrement } from '../actions';
 
 class Counter extends Component {
-    incrementIfOdd = () => {
+    constructor(props) {
+        super(props);
+    }
+
+    incrementIfOdd = (e) => {
         // Stretch Problem: Implement an increment function that
         // only increments if the counter value is odd
+        console.log('increment if odd was triggered');
+        if (this.props.count % 2 === 1) {
+            this.props.increment()
+        }
     };
 
-    incrementAsync = () => {
+    incrementAsync = (e) => {
```

Comment:
Since you aren't utilizing the event, you don't need to pass it in the parameters here